framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2653,6 +2653,58 @@ public AnnotatedWildcardType getUninferredWildcardType(AnnotatedTypeVariable typ
         return wctype;
     }
 
+    /**
+     * Returns the most specific base type of {@code annotatedTypeMirror} whose erasure type
+     * is the upper bound of {@code wildcard}, if {@code wildcard}'s upper
```

Comment:
What do you mean with "whose erasure type is the upper bound"?
Is the "is" supposed to mean "equals"? In the example below, the erasure of "Nullable String" is "Nullable String", which isn't equal to the upper bound of the wildcard (NonNull Object). Is that the reason why nothing happens in the example?